README.md

```diff
@@ -155,6 +179,11 @@ Logspout relies on the Docker API to retrieve container logs. A failure in the A
 * `SYSLOG_STRUCTURED_DATA` - datum for structured data field
 * `SYSLOG_TAG` - datum for tag field (default `{{.ContainerName}}+route.Options["append_tag"]`)
 * `SYSLOG_TIMESTAMP` - datum for timestamp field (default `{{.Timestamp}}`)
+* `MULTILINE_ENABLE_DEFAULT` - enable multiline logging for all containers (default `true`)
```

Comment:
Per @josegonzalez comment, can you clarify this to mean that the default is true only if you're using the multiline adapter?

Reply:
something like "enable multiline logging for all containers when using the multiline adapter (default `true`)". might seem redundant but this documentation is getting a bit hairy and i'd rather over clarify than not.